### PR TITLE
Replace deleted pages with link to new pages

### PR DESCRIPTION
**Pull request by fvisin**
Fix external broken links, as suggested by @nouiz in #3924

### CODE REVIEW

**Comment by abergeron** [comment]
Apparently your syntax is bad.  Travis has this to say:
```
/home/travis/build/Theano/Theano/doc/tutorial/extending_theano.txt:2: WARNING: Field list ends without a blank line; unexpected unindent.
```

**Comment by fvisin** [comment]
Ok....before pushing I checked locally and everything was working, I don't get why the syntax is wrong but I tried something and pushed it. Let's see if Travis complains.

**Comment by nouiz** [comment]
still this travis error.

/home/travis/build/Theano/Theano/doc/tutorial/extending_theano.txt:2: WARNING: Field list ends without a blank line; unexpected unindent.

File: doc/tutorial/extending_theano.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+:orphan:
+This page has been moved. Please refer to: :ref:`extending_theano`.
```
Comment (nouiz) [inline doc/tutorial/extending_theano.txt]: Maybe he don't like the fact that there is no title?

Which version of sphinx do you have installed? Maybe if the version installed by travis is older, we could update it?

Maybe it the the first : from "to:" that he don't like? Or the orphan keyword?

Comment (abergeron) [inline doc/tutorial/extending_theano.txt]: I think it wants a newline after the `:orphan:`.

**Comment by fvisin** [comment]
Fixed! Can be merged finally.

**Comment by nouiz** [comment]
thanks

**Pull request merged**